PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
The original author's info should be kept.
Ideally, you add your name & the updating date after to keep file being tracked.